src/main.cpp

```diff
@@ -47,6 +47,7 @@ bool parse_config(char * config_file, std::string * port) {
 
   *port = get_port(config);
   if (*port == "") {
+    std::cout << "Please specify a port.\n";
```

Comment:
Maybe should be cerr? I think that's where we're directing all of the other error-related statements in this file (could be mistaken)